include/ignition/gazebo/gui/GuiRunner.hh

```diff
@@ -20,6 +20,7 @@
 #include <ignition/msgs/serialized.pb.h>
 
 #include <QtCore>
+#include <memory>
 #include <string>
 
 #include <ignition/transport/Node.hh>
```

Comment:
You can possibly move this `include` and the next one to the `.cc`.

Reply:
True! Moved in 66a2192